Scripts/Services/LootGeneration/RunicReforging/RunicReforging.cs

```diff
@@ -1981,18 +1968,19 @@ public static void GenerateRandomItem(Item item, Mobile killer, int basebudget,
                     {
                         mods = Math.Max(1, GetProperties(5));
 
-                        perchigh = Math.Max(50, Math.Min(500, budget) / mods);
-                        perclow = Math.Max(20, perchigh / 3);
+                        perchigh = Math.Max(50, Math.Min(550, budget) / mods);
+                        perclow = Math.Max(20, Convert.ToInt32(perchigh / 3 * perc));
+                        budget = IsPowerful(basebudget) ? basebudget : budget;
```

Comment:
If the baseBudget (from the fame calculations), is greater than the current budget, then the budget will increase. This allows higher level mobs to drop single property items overcapped.